cmd/login.go

```diff
@@ -119,8 +119,12 @@ Example:
 			if err != nil {
 				return err
 			}
+
 			Config.Token = refreshToken
 		}
+
+		TokenStore.Clean()
```

Comment:
Actually I would do that. If somebody copy-pasted a config file as a backup and this will delete old token store and the new and old one will use the same token store.

At the beginning of the login, I would create a new token store like this:
		```
tmpfile, tempFileError := ioutil.TempFile("", "tokenstore")
		if tempFileError != nil {
			logging.Error("Token Store file can not be read due to error: %v\n", tempFileError)
		}
		Config.TokenStorePath = tmpfile.Name()
	TokenStore = &client.FileBackedTokenStore{
		Path: Config.TokenStorePath,
	}
```